### PR TITLE
Fix basic query tag category and value dropdown

### PR DESCRIPTION
**Pull request by bdunavant**

### CODE REVIEW

File: src/datasource.ts [hunk 1]
```diff
@@ -557,18 +559,41 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
     return this.irondbSimpleRequest('GET', queryUrl, false, true);
   }
 
-  metricTagValsQuery(metricQuery: string, cat: string, from?: number = null, to: number = null) {
-    let queryUrl = '/find' + this.getAccountId() + '/tag_vals?category=' + cat + '&query=';
-    queryUrl = queryUrl + metricQuery;
+  // Used by metricFindQuery
+  metricFindTagValsQuery(metricQuery: string, cat: string, from?: number = null, to: number = null) {
+    let queryUrl = '/find' + this.getAccountId() + '/tag_vals?category=' + cat + '&query=' + metricQuery;
```
Comment (graefen) [inline src/datasource.ts:564]: Is this supposed to be `/find123` or does `this.getAccountId()` return a leading slash? If not, this looks incorrect.

Comment (bdunavant) [reply src/datasource.ts:564]: getAccountId() {
    return this.irondbType === 'standalone' ? '/' + this.accountId : '';
  }

File: src/datasource.ts [hunk 2]
```diff
@@ -557,18 +559,41 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
...
+      queryUrl += '&activity_start_secs=' + _.toInteger(from);
+      queryUrl += '&activity_end_secs=' + _.toInteger(to);
+    }
+    log(() => 'metricFindTagValsQuery() queryUrl = ' + queryUrl);
+    return this.irondbSimpleRequest('GET', queryUrl, false, true, false);
+  }
+
+  metricTagValsQuery(encodedMetricName: string, cat: string, from?: number = null, to: number = null) {
+    let queryUrl =
+      '/find' + this.getAccountId() + '/tag_vals?category=' + cat + '&query=and(__name:' + encodedMetricName + ')';
```
Comment (graefen) [inline src/datasource.ts:576]: same as above

File: src/datasource.ts [hunk 3]
```diff
@@ -557,18 +559,41 @@ export default class IrondbDatasource extends DataSourceApi<IrondbQueryInterface
...
+      log(() => 'metricTagsValsQuery() activityWindow = [' + from + ',' + to + ']');
       queryUrl += '&activity_start_secs=' + _.toInteger(from);
       queryUrl += '&activity_end_secs=' + _.toInteger(to);
     }
     log(() => 'metricTagValsQuery() queryUrl = ' + queryUrl);
     return this.irondbSimpleRequest('GET', queryUrl, false, true, false);
   }
 
+  metricTagCatsQuery(encodedMetricName: string, from?: number = null, to: number = null) {
+    let queryUrl = '/find' + this.getAccountId() + '/tag_cats?query=and(__name:' + encodedMetricName + ')&query=';
```
Comment (graefen) [inline src/datasource.ts:587]: same as above

**Comment by graefen** [comment]
I don't know the codebase, but the syntax looks correct :)

**Pull request merged**